imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/menu/UpdateImage.java

```diff
@@ -67,8 +68,7 @@ public CommandResponse call() throws Exception {
             if (fromImage == null || fromImage.isEmpty()) {
                 return new CommandResponse(-1, "update requires a base image. use --fromImage to specify base image");
             } else {
-                cmdBuilder.add(Constants.BUILD_ARG);
```

Comment:
Are we still using the (runtime) - build-arg to override the FROM image in the docker file ?

Reply:
hmm, I don't see that line in my file